src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -1,38 +1,20 @@
-import React, { useEffect, useState } from "react";
+import React from "react";
 import { TableGrid } from "components/attendee/TableGrid";
+import { useBackgroundGradient } from "components/attendee/useBackgroundGradient";
 
-import {
-  ALWAYS_EMPTY_ARRAY,
-  CONVERSATION_TABLES,
-  DEFAULT_VENUE_LOGO,
-  PORTAL_INFO_ICON_MAPPING,
-} from "settings";
+import { CONVERSATION_TABLES } from "settings";
 
 import { GenericVenue } from "types/venues";
-import { VenueTemplate } from "types/VenueTemplate";
 
 import { WithId } from "utils/id";
 
-import { useAnalytics } from "hooks/useAnalytics";
-import { useExperience } from "hooks/useExperience";
-import { useRelatedVenues } from "hooks/useRelatedVenues";
-import { useShowHide } from "hooks/useShowHide";
-import { useUpdateTableRecentSeatedUsers } from "hooks/useUpdateRecentSeatedUsers";
 import { useUser } from "hooks/useUser";
 
-import { InformationLeftColumn } from "components/organisms/InformationLeftColumn";
 import { RenderMarkdown } from "components/organisms/RenderMarkdown";
-import { Room } from "components/organisms/Room";
 
-import { InformationCard } from "components/molecules/InformationCard";
-import { TableHeader } from "components/molecules/TableHeader";
-import { TablesControlBar } from "components/molecules/TablesControlBar";
-import { UserList } from "components/molecules/UserList";
+import { Loading } from "components/molecules/Loading";
 
-import { BackButton } from "components/atoms/BackButton";
-import { VenueWithOverlay } from "components/atoms/VenueWithOverlay/VenueWithOverlay";
-
-import "./ConversationSpace.scss";
+import styles from "./ConversationSpace.module.scss";
```

Comment:
The file says it's been moved, but it's also empty. Why not simply delete it and remove this line until there is actual styles to add?

Reply:
forgot to add the file. 🤦🏼‍♀️ 
anyway, moved the styles to the SpaceInfo.module.scss: https://github.com/sparkletown/sparkle/pull/2910/commits/afe06cdd59c668832f14435166c1d3c97fba4064